ci/scripts/check-no-bazelrc-site.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Checks that .bazelrc-site file has not been accidentally committed
+# This file should not be merged upstream
```

Comment:
Super nit: Delimit sentences with period :) 

It sounds like we're referring to the lowRISC/opentitan repo as "upstream" which I found a little confusing.